app.js

```diff
@@ -184,7 +185,7 @@ async function sendRequest(agg_x, dataset_code, selected_band, image_scale, star
 
 }
 
-async function getGeostatsData() {
+async function getGeostatsData(offset) {
```

Comment:
Inside the catch and else block, it will return `"0"` which is a valid value to pass into DIVA Protocol. The risk is that if something fails, then 0 may be passed as the settlement value. Best is to fallback to a value that the function `setFinalReferenceValue` wouldn't accept, such as "n/a" or "-1"